UserShiftsApiService/UserShiftsApiService/Entities/RequestedShiftEntity.cs

```diff
@@ -0,0 +1,10 @@
+namespace UserShiftsApiService.Entities;
+
+public class RequestedShiftEntity
+{
+    public string Id { get; set; }
+    public virtual string UserShiftsRequestId { get; set; }
```

Comment:
we only set the entity which we can `include` as virtual. the id actually exists as a col